### DISCUSSION TITLE
how to Force an object type into a string field

### DISCUSSION THREAD

**Discussion by frankbaozun** [vectordotdev/vector#19564] [Q&A]
### A note for the community

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

**Comment by jszwedko** [comment]
Hi @frankbaozun ! I converted this to a GitHub Discussion. How would you like to represent the object as a string? If you want it as JSON you can use the `encode_json` function.

**Reply by frankbaozun** [reply]
ok! thks.
